README.md

```diff
@@ -74,6 +76,8 @@ Version consists of four numbers:
 ### Contributing
 
 See [Contributing Guidelines](CONTRIBUTING.md) before you leave any comment.
+This project exists thanks to all the people who contribute.
+<a href="graphs/contributors"><img src="https://opencollective.com/midorirb/contributors.svg?width=890" /></a>
```

Comment:
It seems that this is a broken link(?).

Reply:
I think it is because the original repo was listed as dsh0416/midori.rb and now the repo is midori.rb/midori.rb. I've brought this to the attention of someone else who should be able to address it. I don't know how to change the pointer.